src/components/ScaffoldVuer.vue

```diff
@@ -484,6 +485,8 @@ export default {
         external: false,
       },
       fileFormat: "metadata",
+      searchIndex: undefined,
```

Comment:
searchIndex is not reactive, preferably remove this and move it over to beforeCreate.